src/deepsparse/transformers/haystack/nodes.py

```diff
@@ -320,6 +306,25 @@ def load(*args, **kwargs):
             "DeepSparse Engine does not support loading from files"
         )
 
+    def _document_to_passage_input(self, document: Document) -> str:
+        # Optionally appends title
+        #
+        # :param document: document to turn into raw text input
+        # :return: raw text input of document title and content
```

Comment:
```suggestion
        """
        Optionally appends title

        :param document: document to turn into raw text input
        :return: raw text input of document title and content
        """
```

Reply:
Because this is a private method, we use # to avoid the docstring being picked up by sphinx (at least that's what I've been taught)

Reply:
Oh hmm I think sphinx will avoid things prepended with `_`? IDEs won't show the docstring to devs if you use `#` though. Is this a NM thing?

Reply:
we've hd some issues in the past - regardless - we standardize to not including pydoc strings in 'private' functions